<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTestResult</class>
 <widget class="QWidget" name="FormTestResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试结果预览</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tabThroughput">
        <attribute name="title">
         <string>吞吐量测试结果</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="gBoxThroughput">
           <property name="title">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDelay">
        <attribute name="title">
         <string>时延测试结果</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGroupBox" name="gBoxDelay">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="labelDelayType">
                <property name="font">
                 <font>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>时延分类</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidgetDelay">
                <column>
                 <property name="text">
                  <string>帧大小</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>速率</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>媒体类型</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>延迟值</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabBackToBack">
        <attribute name="title">
         <string>背靠背测试结果</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="gBoxBackToBack">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_36">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidgetBackToBack">
              <column>
               <property name="text">
                <string>帧大小</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>类型</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>平均速率</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>平均值</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>最大值</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabLossPackgeRate">
        <attribute name="title">
         <string>丢包率测试结果</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="gBoxLossPackgeRate">
           <property name="title">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>结果路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditFilePath">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BtnOutPut">
         <property name="text">
          <string>输出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
